.github/workflows/publish_private_package.yaml

```diff
@@ -0,0 +1,98 @@
+# Purpose: Publish nightly to a a private gallery just to make
+# sure that the code can be published.  This is like a
+# smoke test.
+
+name: Publish Private Package
+
+on:
+  schedule:
+    - cron: "23 0 * * *" # Execute each day at 00:23 UTC
+  workflow_dispatch:
+  push:
+    paths:
+      - ".github/workflows/publish_private_package.yaml"
+      - "utils/DeployUtils.ps1"
```

Comment:
They are probably helpful, just in case someone else edits DeployUtils, but I commented them out and added a comment that they were helpful for debugging.